tmc-langs-r/src/test/resources/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.Rhistory
```

Comment:
This is a good addition. However, it looks like you have added an empty file `tmc-langs-r/src/test/resources/example_json/.Rhistory` that should've been ignored.

Ingored files can always be deleted with:
```sh
git rm --cached <path>
```